src/libraries/System.Net.Security/tests/FunctionalTests/TlsFrameHelperTests.cs

```diff
@@ -23,11 +23,14 @@ public void SniHelper_InvalidData_Fails(int id, byte[] clientHello)
             InvalidClientHello(clientHello, id, shouldPass: false);
         }
 
-        [Theory]
-        [MemberData(nameof(InvalidClientHelloDataTruncatedBytes))]
-        public void SniHelper_TruncatedData_Fails(int id, byte[] clientHello)
+        [Fact]
+        public void SniHelper_TruncatedData_Fails()
         {
-            InvalidClientHello(clientHello, id, shouldPass: false);
+            // moving inside one test because there are more than 3000 cases and they overflow subresults
+            foreach ((int id, byte[] clientHello) in InvalidClientHelloDataTruncatedBytes())
+            {
+                InvalidClientHello(clientHello, id, shouldPass: false);
```

Comment:
So now if one of the test fails we won't know which one and also the rest of them won't get executed.
If we're fine with this, no problem. I'm just making sure all the side-effects are understood and agreed on.

Reply:
Thank you for pointing this out. I may add additional info to identify failed test, if it helps - we still have an `id`.